requirements-core.txt

```diff
@@ -17,7 +17,7 @@ pycocotools>=2.0.0,!=2.0.2; platform_system != "Windows"
 
 pycocotools-windows; platform_system == "Windows"
 PyYAML>=5.3.1
-scikit-image>=0.15.0
+scipy
```

Comment:
Do we need a version restriction?

Reply:
Surprisingly, no, because Datumaro only uses a single SciPy function, `scipy.linalg.orth`, which was added before Python 3 even existed. :-)